pkg/api/v1/types.go

```diff
@@ -361,6 +381,10 @@ type VirtualMachineInstanceMigrationState struct {
 	Completed bool `json:"completed,omitempty"`
 	// Indicates that the migration failed
 	Failed bool `json:"failed,omitempty"`
+	// Indicates that the migration has been requested to abort
+	AbortRequested bool `json:"abortRequested,omitempty"`
+	// Indicates that the migration has been requested to abort
+	AbortStatus MigrationAbortStatus `json:"abortStatus,omitempty"`
```

Comment:
Are you sure that we need this? The migration will be aborted, we should write an event which says "abort succeed" or "migration succeeded". Then instead of setting this, we should remove the finalizer from the migration object, which makes the object disappear.

Reply:
Same here, was mentally on the wrong object. All good.

Reply:
Yea, events are being sent regardless and the finalizer is being removed once the migration object IsFinal().
I've added these to follow up on @davidvossel's review requesting to add a field to the MigrationState that indicates the status of the cancellation - whether it was successful or not.

I suppose I can check the events, but it may not be as obvious as a status field.